src/style.css

```diff
@@ -58,6 +58,188 @@
     position: relative;
 }
 
+.middle-segment{
+    background-color: black;
+     height: 130vh; 
+    margin-top: -4%;
+    border-top-left-radius: 30px;
+    border-top-right-radius: 30px;
+    position: sticky;
+    z-index: 11;
+}
+
+.midsegment-top{
+    color: white;
+    display:grid ;
+    grid-template-columns: 3fr 1.5fr;
+    height: 7vh;
+}
+
+
+.top-heading{
+    display: flex;
+    padding: 0 10% 0 55%; 
+    font-size: 1.8vw;
+    color: rgb(255 255 255 / 70%);
+    align-items: center;
+    align-content:center ;
+    font-weight: 5;
+}
+
+.top-icons{
+    padding: 2vh 2vw;
+    display:grid ;
+    grid-template-columns: 1fr 1fr 1fr 2fr 1fr;
+    align-items: end;
+    justify-content: space-evenly;
+}
+
+#sunny-ic{
+    border-bottom: 2px solid rgb(8  196 243);
+    padding:2px;
+}
+
+#display-top{
+    display: flex;
+     align-self: center;
+     color: rgb(255 255 255 / 70%);
+    font-size: clamp(1vw ,1.1vw, 2vw); 
+    margin: 0.5vh;
+}
+
+#select-icon{
+    display: flex; 
+    background-color: rgb(255 255 255 / 15%);
+     width: 30px;
+     height: 13px;
+     color: rgb(255 255 255 );
+    align-self:center;
+}
+
+input[type="number"]::-webkit-inner-spin-button,
+input[type="number"]::-webkit-outer-spin-button
+{
+    opacity:1;
+    background-color: rgb(255 255 255 / 50%);
+
+}
+
+
+.arrow-ic{ 
+     width: 1vw;
+    height: 1vh;
+}
+
+.scroll-itemshorz{
+       margin-top: 3vh;
+       box-sizing: border-box;
+       height:40vh;
+       width: 100%;
+        padding-left: 0;
+        padding-right: 0;
+       display: grid;
+       grid-template-columns: 0.5fr 5fr 0.5fr;
+}
+
+.left-arrow,.right-arrow{
+       display: flex;
+       align-items: center;
+        justify-content: center;
+        justify-items: center;
+        justify-self: center;
+       height: 85%;
+       width: 3vw;
+       align-content: center;
+}
+
+.left-arr{
+    width: 8vw; 
+    height: 8vh;
+}
+   
+.left-arrow{
+       position: relative;
+       animation: scrollitemsanimarrowleft 3s 1;
+       animation-timing-function: ease;
+}
+   
+.right-arrow{
+         position: relative;
+         animation: scrollitemsanimarrowright 3s 1;
+         animation-timing-function: ease;
+}
+
+.items-container{
+       display: flex;
+       justify-content: space-between;
+       width: 100%;
+       overflow-x: auto;
+       white-space: nowrap;
+}
+
+::-webkit-scrollbar{
+       width: 0;
+       height: 0;
+       background: transparent;
```

Comment:
Let's not do this, we can customize the scroll bar but it needs to be visible to show the page has contents below